rust/src/ssh/logger.rs

```diff
@@ -35,6 +35,18 @@ fn log_ssh(tx: &SSHTransaction) -> Option<Json> {
                 &tx.cli_hdr.swver,
             );
         }
+        if tx.cli_hdr.hassh.len() > 0 {
+            cjs.set_string_from_bytes(
+                "hassh",
```

Comment:
Is not it a big field to log unconditionally ?

Reply:
I think, it's ok to log hassh parameters if hassh is enabled in `app-layer.protocols.ssh.hassh`.

Reply:
ok